.dev/up_deps.jl

```diff
@@ -0,0 +1,37 @@
+#=
+A simple script for updating the manifest
+files in all of our environments.
+=#
+
+root = dirname(@__DIR__)
+dirs = (
+    root,
+    joinpath(root, "artifacts"),
+    joinpath(root, "test"),
+    joinpath(root, ".dev"),
+    joinpath(root, "perf"),
+    joinpath(root, "docs"),
+    joinpath(root, "experiments/AMIP/modular"),
```

Comment:
@LenkaNovak , I added the environments that I used to update in the latest updates. Let me know if they look good. I tested the script locally and works fine.